lms/static/js/fixtures/search_form.html

```diff
@@ -0,0 +1,7 @@
+<div id="courseware-search-bar" class="search-container">
+    <form role="search-form">
+        <input type="text" class="search-field"/>
```

Comment:
@mattdrayer, @chrisndodge, @cdodge, this form input has no accessible label.  Should be 
```<label for="course-search-input" class="sr">Course Search</label><input id="course-search-input" type="text" class="search-field"/>```